### PR TITLE
RDK-45463 : Update Maintenance Manager to use rfcMgr binary

### PR DESCRIPTION
**Pull request by viveksinghnarwaria**
Test Procedure: Refer the ticket for test procedure
Risks: High
Priority: P1

### CODE REVIEW

File: MaintenanceManager/MaintenanceManager.cpp [hunk 1]
```diff
@@ -246,9 +251,13 @@ namespace WPEFramework {
...
 #endif
-		"RFCbase.sh",
+#if defined(ENABLE_RDKVRFC_RDKE)
+             "rfcMgr",
+#else
+           "RFCbase.sh",
+#endif
 #if defined(ENABLE_RDKVFW_RDKE)
-	    "rdkvfwupgrader",
+	          "rdkvfwupgrader",
```
Comment (PaviP98) [inline MaintenanceManager/MaintenanceManager.cpp:260]: Please fix indentation here

**Review by PaviP98 — APPROVED** [review]
LGTM

File: MaintenanceManager/MaintenanceManager.cpp (hunk 1 above)
Comment (viveksinghnarwaria) [reply MaintenanceManager/MaintenanceManager.cpp]: done

File: MaintenanceManager/MaintenanceManager.cpp [hunk 2]
```diff
@@ -502,6 +511,31 @@ namespace WPEFramework {
...
+                    else if (tasks[i] == compare_strings[5])
+		    {
+                        char buff[1024] = { '\0' };
+
+                        FILE* pipe2 = v_secure_popen("r", "/usr/bin/rfcMgr %s", "&");
+                        FILE *fp2 = fopen("/opt/logs/rfcscript.log", "a");
+                        LOGINFO("Waiting to unlock.. [%d/%d]",i+1,tasks.size());
+                        task_thread.wait(lck);
+
+                        if (pipe2 && fp2)
```
[the remark this answers is not in the document]
Comment (viveksinghnarwaria) [reply MaintenanceManager/MaintenanceManager.cpp]: added NULL Pointer Check

**Review by vinupalackal — APPROVED** [review]

**Pull request merged**